lib/registration.pm

```diff
@@ -199,7 +199,8 @@ sub fill_in_registration_data {
                     sleep 5;
                 }
                 # No license agreements are shown in SLE 15 at the moment
-                if (sle_version_at_least('15') && check_screen([qw(ext-module-registration-codes inst-addon)], 0)) {
+                #if (sle_version_at_least('15') && check_screen([qw(ext-module-registration-codes inst-addon)], 0)) {
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

Reply:
I'm still working on it. Extra checking for those needles will lead to 'else', which is wrong.